pkgs/by-name/em/emocli/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, nimPackages, musl, fetchurl, fetchFromGitLab }:
+
+nimPackages.buildNimPackage {
+  pname = "emocli";
+  version = "1.0.0";
+  src = fetchFromGitLab {
+    owner = "AsbjornOlling";
+    repo = "emocli";
+    rev = "ec2d385b0572ad0b763f8b831284310393ac83c4";
+    hash = "sha256-yJu+8P446gzRFOi9/+TcN8AKL0jKHUxhOvi/HXNWL1A=";
+  };
+  nativeBuildInputs = [ musl ];
+  nimFlags = [
+    "-d:release"
+    "--gcc.exe:musl-gcc"
+    "--gcc.linkerexe:musl-gcc"
+    "--passL:-static"
+    "--maxLoopIterationsVM:1000000000"
+  ];
+  doCheck = true;
+  EMOCLI_DATAFILE = fetchurl {
+    url = "https://unicode.org/Public/emoji/15.1/emoji-test.txt";
+    sha256 = "sha256-2HbuJJqijqp2z6bfqnAoR6jROwYqpIjUZdA5XugTftk=";
+  };
+  meta = with lib; {
+    homepage = "https://gitlab.com/AsbjornOlling/emocli";
+    description = "The emoji picker for your command line";
+    license = licenses.eupl12;
+    maintainers = with maintainers; [ asbjornolling ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ asbjornolling ];
    mainProgram = "emocli";
```

Reply:
Looks like I missed that tag. Thanks.
Fixed :)